docs/grammar/BallerinaParser.g4

```diff
@@ -463,6 +471,7 @@ expression
     |   lambdaFunction                                                      # lambdaFunctionExpression
     |   LEFT_PARENTHESIS typeName RIGHT_PARENTHESIS expression              # typeCastingExpression
     |   LT typeName GT expression                                           # typeConversionExpression
+    |   TYPEOF builtInTypeName                                              # typeAccessExpression
```

Comment:
Why only built-in types?
Are we not going to support structs and any user defined types?